### PR TITLE
kv, store: refine some comments

### PR DESCRIPTION
**Pull request by hanfei1991**

### CODE REVIEW

File: kv/union_store.go [hunk 1]
```diff
@@ -97,6 +97,7 @@ func (it invalidIterator) Value() []byte {
 
 func (it invalidIterator) Close() {}
```
Comment (zimulala) [inline kv/union_store.go]: Remove this blank line

File: store/tikv/snapshot.go [hunk 2]
```diff
@@ -36,7 +36,7 @@ const (
 	batchGetSize  = 5120
 )
 
-// tikvSnapshot implements MvccSnapshot interface.
+// tikvSnapshot implements kv.Snapshot interface.
```
Comment (queenypingcap) [inline store/tikv/snapshot.go]: kv.Snapshot interface -> the kv.Snapshot interface

File: kv/union_store.go [hunk 3]
```diff
@@ -97,6 +97,7 @@ func (it invalidIterator) Value() []byte {
...
 
+// lazyMemBuffer wraps a MemBuffer which is only inited when we modify it.
```
Comment (queenypingcap) [inline kv/union_store.go]: -> lazyMemBuffer wraps a MemBuffer which is to be initialized only when it is modified.

**Comment by hanfei1991** [comment]
@queenypingcap PTAL

**Comment by shenli** [comment]
LGTM
@disksing @tiancaiamao PTAL

File: kv/memdb_buffer.go [hunk 4]
```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/pingcap/tidb/terror"
 )
 
+// memDBBuffer implements MemBuffer interface.
```
Comment (queenypingcap) [inline kv/memdb_buffer.go]: MemBuffer interface -> the MemBuffer interface

**Comment by disksing** [comment]
LGTM.

**Pull request merged**